source/builtin_macros/src/syntax.rs

```diff
@@ -3974,6 +3974,61 @@ pub(crate) fn rewrite_expr(
     proc_macro::TokenStream::from(new_stream)
 }
 
+struct Stmts(Vec<Stmt>);
+
+impl Parse for Stmts {
+    fn parse(input: ParseStream) -> syn_verus::Result<Self> {
+        let mut stmts = Vec::new();
```

Comment:
You may be able to use https://docs.rs/syn/latest/syn/struct.Block.html#method.parse_within here instead, which would parse a little more accurately.